package.json

```diff
@@ -12,7 +12,7 @@
     "test-ci": "./bin/start-test-server & concurrently --success --raw 'npm test' 'npm run test-cypress'",
     "build": "gulp clean && gulp css && webpack",
     "build-production": "export NODE_ENV=production && npm run build && gulp manifest",
-    "build-autotrack": "autotrack -o public/autotrack/autotrack.js -p outboundLinkTracker,eventTracker,maxScrollTracker",
+    "build-autotrack": "npx autotrack -o public/autotrack/autotrack.js -p outboundLinkTracker,eventTracker,maxScrollTracker",
```

Comment:
We commit the `public/autotrack/autotrack.js` file to the repo so we don't need to install `autotracker` every time. Instead we can use `npx` to install it on demand if/when we need to rebuild this file.